packages/flutter_tools/templates/package/README.md.tmpl

```diff
@@ -1,14 +1,36 @@
-# {{projectName}}
+<!-- 
+This README describes the package. If you publish this package to pub.dev,
+then this README supplies most of the contents of the package page.
 
-{{description}}
+For information about how to write a good package README, see the guide for
+[writing package pages](https://dart.dev/guides/libraries/writing-package-pages). 
 
-## Getting Started
+For general information about developing packages, see the Dart guide for
+[creating packages](https://dart.dev/guides/libraries/create-library-packages)
+and the Flutter guide for
+[developing packages and plugins](https://flutter.dev/developing-packages). 
+-->
 
-This project is a starting point for a Dart
-[package](https://flutter.dev/developing-packages/),
-a library module containing code that can be shared easily across
-multiple Flutter or Dart projects.
+TODO: Put a short description here that helps potential users know whether
+this package might be useful for them.
 
-For help getting started with Flutter, view our 
-[online documentation](https://flutter.dev/docs), which offers tutorials, 
-samples, guidance on mobile development, and a full API reference.
+## Features
+
+TODO: List what your package can do. Maybe include images or videos.
+
+## Getting started
+
+TODO: List prerequisites and provide or point to information on how to
+start using the package.
+
+## Usage
+
+TODO: Include useful examples for package users.
```

Comment:
Here we could also recommend linking into `/example` for those that need slightly longer examples.